var/spack/repos/builtin/packages/gptune/package.py

```diff
@@ -303,4 +300,4 @@ def test_scalapack(self):
                     raise SkipTest("Package must be installed with +superlu+mpispawn")
                 with working_dir(join_path(test_dir, app)):
                     terminate_bash_failures(".")
-                    self.bash("run_examples.sh")
+                    self.cmd["bash"]("run_examples.sh")
```

Comment:
This package should probably depend on `bash`. It's not guaranteed to be on the system.